homework/01LessonOne.js

```diff
@@ -105,7 +107,8 @@ var notNotTrue;
 //
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: if a variable has && it is "and" which means both have to be true for it to be
+// true:  || is "or" and means either one thing has to be true for it to be true.
```

Comment:
Good job on understanding the basic logic, @CodingGabe.

But **variable** isn't quite the right word here. We're talking about [boolean expressions](https://en.wikipedia.org/wiki/Boolean_expression).